test/policy_ordering.go

```diff
@@ -78,19 +96,21 @@ func PolicyOrdering(labels ...string) bool {
 				DoRootComplianceTest(policyWithDepName, policiesv1.NonCompliant)
 			})
 			It("Should become pending again when the initial policy is deleted", func() {
+				cleanEvent()
 				DoCleanupPolicy(initialPolicyYaml)
 				DoRootComplianceTest(policyWithDepName, policiesv1.Pending)
 			})
 			AfterAll(cleanup)
 		})
-		Describe("Ordering via an extraDependency on a ConfigurationPolicy", func() {
+		Describe("Ordering via an extraDependency on a ConfigurationPolicy", Ordered, func() {
 			BeforeAll(func() {
 				By("Creating the initial policy to use as a dependency")
 				DoCreatePolicyTest(initialPolicyYaml, GvrConfigurationPolicy)
 				DoRootComplianceTest(initialPolicyName, policiesv1.NonCompliant)
 
 				By("Creating the policy that depends on the initial policy")
 				DoCreatePolicyTest(policyWithExtraDepYaml)
+				cleanEvent()
```

Comment:
Same as the previous comment. I don't think we want this here since you are artificially bringing the policy to pending. I'll defer to @JustinKuli on this though.